src/pages/Home.scss

```diff
@@ -0,0 +1,27 @@
+.back {
+    position: relative;
```

Comment:
Don't use position: relative here use position absolute on the image
Here is an example of how you might center the login box
`
#container {
    display: flex;           /* establish flex container */
    flex-direction: column;  /* make main axis vertical */
    justify-content: center; /* center items vertically, in this case */
    align-items: center;     /* center items horizontally, in this case */
    height: 300px;
}

.box {
    width: 300px;
    margin: 5px;
    text-align: center;     /* will center text in <p>, which is not a flex item */
}
`